### PR TITLE
Implemented Delete Ride Request and Changed Request Form Model

### PR DESCRIPTION
**Pull request by Hoang247**
- Added delete ride request functionality
- Fixed models so ride request forms are based on Ride model instead of RideRequestPost model and deleted RideRequestPost model

- Added href to details button on dashboard

### CODE REVIEW

**Comment by Hoang247** [comment]
Screen Shot 2020-02-29 at 2 16 31 PM
Screen Shot 2020-02-29 at 2 17 20 PM
Here's what some added pages look like. They work, but don't look very nice, so I'll clean it up later

File: homeyess/website/forms.py [hunk 1]
```diff
@@ -60,18 +60,19 @@ class Meta:
...
-	class DateInput(forms.DateInput):
-		input_type = 'date'
-	pickup_date = forms.DateField(widget=DateInput, required=True)
-	interview_duration = forms.CharField(max_length=30, required=True, help_text=' (in minutes)')
-	pickup_address = forms.CharField(max_length=200, required=True)
-	interview_address = forms.CharField(max_length=200, required=True)
-	company_name = forms.CharField(max_length=200, required=True)
+    class DateInput(forms.DateInput):
+        input_type = 'date'
+    pickup_datetime = forms.DateTimeField(required=False, widget=forms.HiddenInput)
```
Comment (aHunsader) [inline homeyess/website/forms.py]: I dont think you need to include these 2 lines of hidden input but I am not sure

**Comment by aHunsader** [comment]
Thank you for the thorough description / images - it is very helpful.

1) can you rename the pr from Hoang to a small description (that way it's easy to look back at commit history and know whats happening)
2) I deleted the ride_status field of the ride model because it was unnecessary. Can you make sure that you do not use it anywhere.

Overall lgtm but we have some serious design work ahead of us lol. Lmk when you make the above changes and Ill merge

**Comment by Hoang247** [comment]
Current create/update request and view request form page
Screen Shot 2020-03-01 at 4 40 17 PM
Screen Shot 2020-03-01 at 4 40 28 PM

File: homeyess/website/models.py [hunk 2]
```diff
@@ -77,17 +77,32 @@ class Ride(models.Model):
...
 	interview_datetime = models.DateTimeField()
 	interview_duration = models.IntegerField()
+	pickup_datetime = models.DateTimeField()
 	interview_address = models.CharField(max_length=200)
 	interview_company = models.CharField(max_length=100)
-	start_datetime = models.DateTimeField(null=True, blank=True)
-	pickup_datetime = models.DateTimeField(null=True, blank=True)
-	end_datetime = models.DateTimeField(null=True, blank=True)
-
+	end_datetime = models.DateTimeField()
```
Comment (aHunsader) [inline homeyess/website/models.py]: please change this section back to how it was in master

**Comment by aHunsader** [comment]
lgtm

**Pull request merged**